mozilla-lockwise/lockwise-android
# 1152: Toast placement for copy, create, delete

Fixes #1152 

Specify each bottom margin for the toast notifications. The `bottomMargin` value set based on the nav location: toasts on the item list will appear above the `+` create button, and other toasts will have the normal bottom margin at the bottom of the screen.

## Testing and Review Notes
* copy username/copy password appear at the bottom of the screen
* delete confirmation appears above the `+` button on the item list
* create confirmation appears on the item detail screen at the bottom (same place as copy)

## Screenshots or Videos
_(save gif tbd on #1163)_

<img width="325" alt="copy_toasts" src="https://user-images.githubusercontent.com/43795363/73658829-55b1ab80-4695-11ea-960c-fbb9910a7a6b.gif">

<img width="325" alt="delete_toast" src="https://user-images.githubusercontent.com/43795363/73658842-5a765f80-4695-11ea-93bf-5f11ce07b2ff.gif">


## To Do

  - optional: consider adding instrumentation (integration/UI) tests
- consider running this branch in a debug simulator and check for memory leak notifications or any warnings
- [ ] check on the [accessibility](https://mozilla-lockwise.github.io/lockwise-android/accessibility/) of any added UI
